### PR TITLE
Update Perl::Tidy to 20230701

### PR DESCRIPTION
**Pull request by perlpunk**

### CODE REVIEW

**Review by kraih — APPROVED** [review]

File: t/develcover.t [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    my $res = -1;
+    return pass 'foo' if $res == -1;
+}
+
+subtest 'color output can be configured via the command-line' => sub {
+    isotovideo();
+    pass "ok";
+};
+
+done_testing();
```
Comment (kraih) [inline t/develcover.t:18]: What are these new tests about? Accidental addition?

Comment (perlpunk) [reply t/develcover.t]: oops! accidentally added local tests I was playing with. fixed

**Review by okurz — APPROVED** [review]
Looks good but we should give some heads-up time for according downstream test distribution changes to be prepared.

**Comment by perlpunk** [comment]
> Looks good but we should give some heads-up time for according downstream test distribution changes to be prepared.

Ok. But note that other os-autoinst PRs will be blocked on this.
https://github.com/os-autoinst/os-autoinst/pull/2343 is failing because of this, because the used container already has the new perltidy version installed.

**Comment by okurz** [comment]
Yes, sure. I already pinged people and "QE Core" has reacted but I don't think they have prepared a PT yet. Ok, I guess 8h need to suffice ;)

**Pull request merged**